cypress/page/products-list.page.ts

```diff
@@ -0,0 +1,21 @@
+class ProductsListPage {
+    private tShirtMenu: string;
+    private proceedToCheckout: string
+
+    constructor() {
+        /*With this selector suggested is possible to have the number 1 as a variable
+        to tests more items in the page*/
+        this.tShirtMenu = "[title='Add to cart'][data-id-product='1']";
```

Comment:
this selector does not belong to the t-shirt Menu, it is pointing to the add to cart button. I expect this Page Object to have two properties, one for the AddToCart button and other for the ProceedToCheckout button. You have the proceed to checkout but you are using the AddToCart selector for the tshirtMenu option that should go on the MenuContentPage class

Reply:
Got it! Also, it solved the issue with the double `goToPayment().`